admin/sections/config/providers/openai.tsx

```diff
@@ -44,10 +46,45 @@ export const OpenAIForm = ({
           </FormItem>
         )}
       />
-      <ModelForm
-        form={form}
-        defaultValues={defaultValues}
-        supportedModels={supportingModels}
+      <div className="flex space-x-4">
+        <div className="w-1/2">
+          <ModelForm
+            form={form}
+            defaultValues={defaultValues}
+            supportedModels={supportingModels}
+          />
+        </div>
+        <div className="w-1/2">
+          <ModelForm
```

Comment:
what's the default for embedding: `text-embedding-3-small`?

Reply:
Yes it's. The default value is already specified in the client/providers